modules/javafx.media/src/main/java/com/sun/media/jfxmediaimpl/platform/gstreamer/GSTMedia.java

```diff
@@ -90,8 +90,8 @@ public synchronized void dispose() {
      * @return A handle to the native peer of the media.
      */
     private native int gstInitNativeMedia(Locator locator,
-                                               String contentType,
-                                               long sizeHint,
-                                               long[] nativeMediaHandle);
+                                          String contentType,
+                                          long sizeHint,
+                                          long[] nativeMediaHandle);
```

Comment:
This file contains whitespace-only changes. Maybe revert the file since it is otherwise unchanged?

Reply:
Reverted.